bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
+import bark, os
+
+
+def main(wav):
+	if not isinstance(wav, str):
+	    raise ValueError('A file path string is required')
+	name, ext = os.path.splitext()   # get the original file name. e.g. ab from ab.wav
```

Comment:
You're missing a parameter to `splitext` - this line as written raises an exception.